pytensor/tensor/subtensor.py

```diff
@@ -722,32 +732,51 @@ def make_node(self, x, *inputs):
                     f"Incompatible types for Subtensor template. Expected {input.type}, got {expected_type}."
                 )
 
-        # infer the broadcasting pattern
-        padded = get_constant_idx(
-            self.idx_list, (None,) + inputs, allow_partial=True
-        ) + [slice(None, None, None)] * (x.type.ndim - len(idx_list))
+        padded = [
```

Comment:
nitpick you may want to call this "enhancement" in the commit prefix. None was a correct static shape output, just not the best we could have done.